src/Polysemy/Resource.hs

```diff
@@ -48,6 +49,20 @@ data Resource m a where
 
 makeSem ''Resource
 
+------------------------------------------------------------------------------
+-- | A variant of 'bracket' where the return value from the first computation
+-- is not required.
+--
+-- cf. 'Control.Exception.bracket' and 'Control.Exception.bracket_'
+--
+-- @since 1.3.1.0
```

Comment:
I think we usually leave this as a `@since TODO` and not change the version `package.yaml`/`.cabal`, because there might not be a `1.3.1.0` release on hackage, making this haddock a bit confusing.
@KingoftheHomeless @TheMatten ?

Reply:
I doubt there's gonna be a `1.3.1.0`, unless we discover something we desperately need to fix.
So yes, leave this as a TODO.

Reply:
No problem, fixed